Makefile

```diff
@@ -6,15 +6,21 @@ EXTRACTED_DJANGOJS := $(EXTRACT_DIR)/djangojs-partial.po
 EXTRACTED_TEXT := $(EXTRACT_DIR)/text.po
 EXTRACTED_TEXTJS := $(EXTRACT_DIR)/textjs.po
 
-requirements:
-	pip install -r requirements.txt
-	pip install -e .
+quality: ## check coding style with pycodestyle and pylint
+	pycodestyle diagnostic_feedback --max-line-length=120
+	pylint diagnostic_feedback --disable=all --enable=function-redefined,undefined-variable,unused-variable
 
-test-requirements:
-	pip install -r test-requirements.txt
+test: ## run tests in the current virtualenv
+	mkdir -p var  # for var/workbench.log
+	python run_tests.py
 
-dev-requirements:
-	pip install -r requirements-dev.txt
+requirements: ## install development environment requirements
+	pip install -r requirements.txt --exists-action w
+	pip install -r requirements-dev.txt --exists-action w
```

Comment:
Shouldn't we keep dev and install requirements separate? Both have a separate set of requirements. For installation in LMS, it should use LMS's provided packages(django etc) and not force it's own

Reply:
yes this will not force its own rather will use platforms packages, it only requires `xblock` package as listed [here](https://github.com/mckinseyacademy/xblock-diagnosticfeedback/pull/33/files#diff-2eeaed663bd0d25b7e608891384b7298R37). These requirements are only for development.